README.md

```diff
@@ -1,26 +1,19 @@
 # BuyTogether
-
 Welcome to our group social project!.
-In this project we have developed a web application which suppose to help people group up
-and organize a large scale food buy.
-Doing so, we "cut" the amount of hands the product go through which will benefit the customer
-with cheaper prices.
-
+In this project, we have developed a web application that is supposed to help people group up,
+and organize a join food purchase.
+Doing so, we remove the middlemen and reduce the price of the food products.
```

Comment:
I think you want an extra line break here. Keep in mind that markdown will add an actual line break on the rendered file only if you add two line breaks on the text. This is to allow you to break line of codes so they can be readable in editors, while letting the output to be rendered according to the window size etc.
```suggestion

Doing so, we remove the middlemen and reduce the price of the food products.
```